Shift3/boilerplate-client-react
# Refactor page styles

## Changes
1. Moved styled components in `features/styles/PageStyles.tsx` to `common/styles`.
    - styled components were further subdivided into modules by type
        1. `common/styles/button`
        2. `common/styles/button`
        3. `common/styles/form`
2. Moved `CustomButton` styled component from `features/auth/components/button/styled.tsx` to `common/styles/button`
3. Updated import paths to reflect new location of components.

## Purpose
The styled components that were moved in this PR are shared between all the features and are not part of any one feature. Thus, it makes sense to move them to a common folder than inside a feature folder.

## Testing
1. Pull in the changes to your local copy of this branch.
2. Start the app using `yarn start`.
3. Check that the following pages still render correctly
    1. login page
    2. sign up page
    3. forgot password page
    4. reset password page
    5. activate account page
    6. agency list view
    7. create agency view
    8. update agency view
    9. agent list view
    10. create agent view
    11. update agent view
    12. update user profile page
    13. change password page
    14. user list view
    15. create user view
    16. update user view
4. Check that the following components still render correctly
    1. top navbar
    2. side navbar
    3. confirmation modals

### Closes #486
